tests/integration/cgroups.bats

```diff
@@ -190,6 +190,22 @@ function setup() {
 	[[ "$output" == *'invalid configuration'* ]]
 }
 
+@test "runc run (blkio weight)" {
+	requires root cgroups_v2
+
+	set_cgroups_path "$BUSYBOX_BUNDLE"
+	update_config '.linux.resources.blockIO |= {"weight": 750}' "${BUSYBOX_BUNDLE}"
+
+	runc run -d --console-socket "$CONSOLE_SOCKET" test_cgroups_v2_blkio
```

Comment:
If you add a new container name you need to add it to the `teardown()` hook at the start -- `teardown_running_container`.

Reply:
I noticed it, too, but did not comment because I am fixing all this soon (after https://github.com/opencontainers/runc/pull/2757 is merged).